framework/ldtest/test_scope.go

```diff
@@ -200,3 +206,17 @@ func (t *T) RequireCapability(name string) {
 		t.SkipWithReason(fmt.Sprintf("test service does not have capability %q", name))
 	}
 }
+
+// Helper marks the function that calls it as a test helper that shouldn't appear in stacktraces.
+// Equivalent to Go's testing.T.Helper().
```

Comment:
The `testify/assert` and `testify/require` functions all call `Helper()` if it's defined, and so will the `matchers` functions once https://github.com/launchdarkly/go-test-helpers/pull/13 is merged.